src/components/LoanCards/Buttons/ActionButton.vue

```diff
@@ -1,35 +1,20 @@
 <template>
 	<component
 		:is="currentButtonState"
-		class="button lend-button"
-		@click.native="addToBasketClick"
-		:id="id" />
+		class="action-button"
+		:loan-id="loanId" />
 </template>
 
 <script>
-import basketItems from '@/graphql/query/basketItems.graphql';
-import basketCount from '@/graphql/query/basketCount.graphql';
-import addToBasket from '@/graphql/mutation/addToBasket.graphql';
 import _includes from 'lodash/includes';
-import LendButton from './LendButton';
+import Lend25Button from './Lend25Button';
 import CheckoutNowButton from './CheckoutNowButton';
 import LendAgainButton from './LendAgainButton';
-import PlaceholderButton from './PlaceholderButton';
+import ReadMoreButton from './ReadMoreButton';
 
 export default {
-	components: {
-		LendButton,
-		CheckoutNowButton,
-		LendAgainButton,
```

Comment:
Nice that we don't have "assign" these to this component and can just pass them through.